web/template/about.gohtml

```diff
@@ -20,12 +20,13 @@
     <div class="border-gray-500 border-t-2 mt-4 mb-4"></div>
     <h1 class="text-1">Übertragung von Veranstaltungen</h1>
     <p>
-    Wir übertragen gern alle Veranstaltungen, die sich an TUM-Studierende oder Mitglieder richten.<br>
-    Hier gibt es zwei Möglichkeiten:
+        Wir übertragen gern alle Veranstaltungen, die sich an TUM-Studierende oder Mitglieder richten.<br>
```

Comment:
There seem to be some commits from another pr, can you merge `dev`?

Reply:
I think it was some stashing issue unrelated to the merge, I already just fixed it locally. I will push it now!